tests/tape_tests/conftest.py

```diff
@@ -28,75 +28,89 @@ def create_test_rows():
 
     return rows
 
-def create_test_column_mapper():
 
+@pytest.fixture
+def create_test_column_mapper():
     return ColumnMapper(id_col="id", time_col="time", flux_col="flux", err_col="err", band_col="band")
 
-def create_test_source_table(npartitions=1):
 
-    return dd.from_dict(
-        create_test_rows(), 
-        npartitions)
+@pytest.fixture
+@pytest.mark.parametrize("create_test_rows", [("create_test_rows")])
```

Comment:
The `@pytest.mark.parametrize` feels extraneous here, but I'm guessing you did what you had to do to get the tests to work =]